src/main/java/tasks/TaskList.java

```diff
@@ -178,20 +187,21 @@ public void addDeadlineTask(String description, String end) {
      * @param start start Date/Time of the Event object.
      * @param end end Date/Time of the Event object.
      */
-    public void addEventTask(String description, String start, String end) {
+    public String addEventTask(String description, String start, String end) {
```

Comment:
'@return' in the Javadocs might be missing because of the recent change to the method return type.